pipelines/gitops-pull-request-rhtap/gitops-pull-request.yaml

```diff
@@ -80,12 +104,66 @@ spec:
           value: $(params.ec-rekor-host)
         - name: TUF_MIRROR
           value: $(params.ec-tuf-mirror)
+      runAfter:
+        - get-images-to-verify
+      taskRef:
+        name: verify-enterprise-contract
+        version: "0.1"
+
+    - name: get-images-to-upload-sbom
       runAfter:
         - gather-deploy-images
+      params:
+        - name: INPUT
+          value: $(tasks.gather-deploy-images.results.IMAGES_TO_VERIFY)
+        - name: JQ_ARGS
+          value:
+            - --compact-output
+            # upload SBOMs for images that get promoted to staging or production
+            - .components | map(select(.env != "development") | .containerImage) | unique
```

Comment:
It ended being a sort of "build your own pipeline". Very easy to mess everything up by changing the jq usage.

But the alternatives were:
- add a task that does the same thing as `gather-deploy-images` but ignores dev images (AFAIK we don't want to upload SBOMs on promotion to dev, only stage and prod)  and outputs result in a different format
- tightly couple the download sbom task to the output format of gather-deploy-images (which is highly EC-specific/Konflux-specific)

Reply:
Got rid of jq by parametrizing the gather-deploy-images task instead and running it twice (once for EC, once for SBOMs)